main.py

```diff
@@ -42,12 +31,30 @@ def get_day_element(driver: WebDriver, day: str) -> WebElement:
     return elem
 
 
-def apply_for_single_day(
-    driver: WebDriver, ballot_day: str, code: str, **kwargs
+def get_session_element(driver: WebDriver, session: str) -> WebElement:
+    """Used for events with Day/Night sessions."""
+    if session == "昼公演":
+        elem = WebDriverWait(driver, 10).until(
+            EC.element_to_be_clickable((By.LINK_TEXT, "＜昼公演＞お申込み"))
+        )
+
+    elif session == "夜公演":
+        elem = WebDriverWait(driver, 10).until(
+            EC.element_to_be_clickable((By.LINK_TEXT, "＜夜公演＞お申込み"))
+        )
+    return elem
+
+
+def apply_for_single_session(
+    driver: WebDriver, session: str, code: str, **ballot_info
 ) -> None:
-    day = get_day_element(driver, ballot_day)
-    print(f"Applying to {ballot_day}")
-    day.click()
+    # TODO: generalize to sessions that aren't "Day.1/Day.2" or "昼公演/夜公演"
+    if "Day." in session:
+        session_button = get_day_element(driver, session)
+    else:
+        session_button = get_session_element(driver, session)
+    print(f"Applying to {session}")
+    session_button.click()
```

Comment:
With the change below to the main loop
This could potentially be simplified to:

```py
def get_session_element(driver: WebDriver, session_name: str) -> WebElement:
    elem = WebDriverWait(driver, 10).until(
        EC.element_to_be_clickable((By.LINK_TEXT, "＜{0}＞お申込み".format(session_name)))
    )

    return elem


def apply_for_single_session(
    driver: WebDriver, session_name: str, code: str, **ballot_info
) -> None:
    session_button = get_session_element(driver, session_name)
    print(f"Applying to {session}")
    session_button.click()
```